### PR TITLE
Prepend http protocol

### PR DESCRIPTION
**Pull request by leandroncbrito**
#8 It's not possible to convert using from-url without protocol http(s)

Added function to check if the URL has HTTP protocol and prepend it.

### CODE REVIEW

File: pdf-service.js [hunk 1]
```diff
@@ -82,6 +82,16 @@ app.get('/from-url', async (req, res) => {
     }
 });
 
+function withHttp (url) {
+    if (!url) {
```
Comment (fabriciojs) [inline pdf-service.js:86]: I think we don't need this check, do we?

**Review by fabriciojs — CHANGES_REQUESTED** [review]
Looks good! Just  one comment.

File: pdf-service.js (hunk 1 above)
Comment (leandroncbrito) [reply pdf-service.js:86]: If we don't have the url parameter it'll return an error in line 90 (
I can change url to "let", remove it by applying the function after url parameter validation, like this:

```javascript
    // change const to let
    let url = req.query.url;
    console.log('/from-url');

    if (!url) {
        let msg = 'missing parameter: \'url\'';
        deliverJson(res, {msg, params: req.params}, 400);
        return;
    }

    let pdfFilePath;
    try {
        // call the withHttp function before send to generatePdf()
        pdfFilePath = await generatePdf(withHttp(url), req.query.media);
```

Comment (leandroncbrito) [reply pdf-service.js:86]: @fabriciojs 
What do you think? I like this new version of code, we check url parameter once

**Review by fabriciojs — APPROVED** [review]

**Pull request merged**